src/systems/apply_link_wrench/ApplyLinkWrench.cc

```diff
@@ -80,16 +80,17 @@ class gz::sim::systems::ApplyLinkWrenchPrivate
 /// \param[in] _msg Entity message. If it's a link, that link is returned. If
 /// it's a model, its canonical link is returned.
 /// \param[out] Force to apply.
+/// \param[out] Offset of the force application point relative to the
```

Comment:
This implies that the offset in `msgs::EntityWrench` and by association, in the `component::ExternalWorldWrench`. is relative to the COM, but the point of application is the link origin. Am I right? If so, I think we should avoid having two different reference points. If the point of application is the link origin, the offset should also be from the link origin.

Reply:
Changed in ee693bc so that the Link API and the ApplyLinkWrench system interpret the offset in the link frame. This way, the semantics of the offset are consistent between them, the `msgs::EntityWrench` message and the `component::ExternalWorldWrench` component.